breathecode/events/urls.py

```diff
@@ -1,14 +1,17 @@
 from django.urls import path
 from .views import (EventView, EventTypeView, EventCheckinView, get_events, eventbrite_webhook,
                     AcademyEventView, AcademyVenueView, ICalCohortsView, ICalEventView, ICalStudentView,
-                    AcademyOrganizationView)
+                    AcademyOrganizationView, OrganizationWebhookView)
 
 app_name = 'events'
 urlpatterns = [
     path('', EventView.as_view(), name='root'),
     path('all', get_events, name='all'),
     path('academy/event', AcademyEventView.as_view(), name='academy_all_events'),
     path('academy/organization', AcademyOrganizationView.as_view(), name='academy_organization'),
+    path('academy/organization/eventbrite/webhook',
+         OrganizationWebhookView.as_view(),
+         name='eventbrite_webhook'),
```

Comment:
the name for the url should be `name='academy_organizarion_eventbrite_weebhook'` because we try write the name with the same structure as the path it belongs